audonnx/core/testing.py

```diff
@@ -89,20 +100,88 @@ def create_model(
 
     # create model
 
-    model = audonnx.Model(
-        graph,
+    model = Model(
+        object,
         transform=transform,
+        device=device,
     )
 
     return model
 
 
-def _identity_graph(
+def create_model_proto(
         shapes: typing.Sequence[typing.Sequence[int]],
-        dtype: int,
-        opset_version: int,
+        *,
+        dtype: int = onnx.TensorProto.FLOAT,
+        opset_version: int = 14,
 ) -> onnx.ModelProto:
-    r"""Create identity graph."""
+    r"""Create test ONNX proto object.
+
+    Creates an identity graph
+    with input and output nodes
+    of the given ``shapes``.
+    For each entry an input and output
+    node will be created.
+    ``-1``, ``None`` or strings
+    define a dynamic axis.
+    Per node,
+    one dynamic axis is allowed.
+    The identity graph can be used as ``path`` argument
+    of :class:`audonnx.Model`.
+
+    Args:
+        shapes: list with shapes defining the input and
+            output nodes of the graph
```

Comment:
This is slightly ambiguous as it could refer to:

* `[input0_shape0, input0_shape1, output0_shape0, output0_shape1]`
* `[[input0_shape0, input0_shape1], [input1_shape0], [output0_shape0, output0_shape1]]`
* `[[node0_shape0], [node1_shape0, node1_shape1]]`

But it is also not easy to phrase, maybe

```rst
        shapes: list with list of dimensional sizes
            defining the identical shapes of the input and output nodes
            of the graph.
            E.g. ``[[2], [1, -1]]``
            would result in a one-dimensional first input and output node of size 2,
            and a two-dimensional second input and output node with a size of 1
            of its first dimension
            and a dynamic size of its second dimension
```

If we change this here, we should also change it for `create_model()`

Reply:
Maybe it's clear that a `shape` is always a list, but it's not clear that you define input and output nodes with the same entry.

Reply:
Ah ok, then maybe:

`list with shapes defining the identical input and output nodes of the graph`

?

Reply:
Ok, changed it accordingly:

![image](https://user-images.githubusercontent.com/10383417/174813168-cb73e2e5-96cc-40ed-85bd-9767c7fd937a.png)

+ updated the description